contributors/guide/contributor-cheatsheet/README.md

```diff
@@ -340,6 +340,11 @@ git checkout master
 git rebase upstream/master
 ```
 
+**Don't forget to `Push` your updated local repo to the remote fork.**
+```
+git push
```

Comment:
Just add this to the above list of commands. I don't think there is a need to explicitly call out a "don't forget" in bold.

Reply:
Sure!